queries/cedar/highlights.scm

```diff
@@ -0,0 +1,62 @@
+; Function calls
+(call_expression
+  function: (selector_expression
+    field: (field_identifier) @function.method))
+
+[
+  "contains"
+  "containsAll"
+] @function
+
+; Identifiers
+[
+  (entity)
+  (principal)
+  (action)
+  (resource)
+  (context)
+  (identifier)
+  (annotation)
+] @variable
+
+(path) @type
```

Comment:
Is this a file path?

Reply:
In Cedar `path` refers to a namespace and a type of an authorization entity. For example, `MyApplication::User` or `Google::Drive::Document`.

Where possible I tried to keep names consistent with the [Cedar grammar](https://docs.cedarpolicy.com/policies/syntax-grammar.html), although this is my first Tree Sitter grammar so I'm unsure if `path` is a reserved word.

Reply:
Nothing is reserved. You might wanna highlight the namespace(s) as `@module` though.

Reply:
Thanks. I added `::` as `@punctuation.delimiter`. For consistency with the existing VS Code plugin (see screenshot below), I've opted to keep the path in place as is. Cedar doesn't have fully fledged namespaces yet, so the namespace is essentially part of the type name.

Thanks for all your fast feedback on this.

<img width="617" alt="image" src="https://github.com/nvim-treesitter/nvim-treesitter/assets/17420369/91c29597-a8b0-49be-9ca4-df039fd405cc">